Changes

```diff
@@ -102,6 +102,10 @@ Working version
   for consistency with OSX & Linux. Previously was located at `lib`.
   (Bryan Phelps, Jordan Walke, review by David Allsopp)
 
+- GPR#2002: Add Format.pp_print_custom_break, a new more general kind of break
+  hint that can emit non-whitespace characters.
+  (Vladimir Keleshev, review by Josh Berdine, Gabriel Radanne, Pierre Weis)
```

Comment:
I think it would be appropriate to also note the design input by Pierre Weis.

Reply:
(If it helps inspiration, on similar occasions I have used the equivalent of "Vladimir Keleshev and Pierre Weis". There is an asymmetry that isn't fully rendered, but I'm sure @keleshev will make many other nice contributions so that being fully precise about each one matters less.)